src/zeit/retresco/testing.py

```diff
@@ -46,10 +50,32 @@ def testSetUp(self):
 product_config = """
 <product-config zeit.retresco>
     base-url http://localhost:[PORT]
+    elasticsearch-url http://tms-backend.staging.zeit.de:80/elasticsearch
```

Comment:
Die Tests laufen alle gegen einen Mock, wir haben diese URL verwendet, um in den Tests durch Umschalten des Layers von `MOCK_ZCML_LAYER` auf `ZCML_LAYER` gegen einen echten Server laufen zu können um die Antworten des Mocks mit der realen Welt abzugleichen.